crates/schema-impl/src/schemas_impl.rs

```diff
@@ -353,16 +357,16 @@ impl SchemasInner {
         if let Some(existing_endpoint) = self.endpoints.get(&endpoint_id) {
             if allow_overwrite {
                 // If we need to overwrite the endpoint we need to remove old services
-                for (svc_name, revision) in &existing_endpoint.services {
+                for svc in &existing_endpoint.services {
                     warn!(
                         restate.service_endpoint.id = %endpoint_id,
                         restate.service_endpoint.address = %endpoint_metadata.address_display(),
                         "Going to remove service {} due to a forced service endpoint update",
-                        svc_name
+                        svc.name
                     );
```

Comment:
I think we should remove this warn logging. We should not warn on normal user operations, the danger is on the user side, not the restate service side.

Reply:
The reason for this log line is that if you manually send discovery requests with force flag, atm we don't have a way to notify back whether you removed a service. At least with log line you notice that something happened :)